troposphere/static/js/components/modals/image_version/ImageVersionEditModal.react.js

```diff
@@ -31,6 +32,10 @@ define(function (require) {
 
       return {
         showOptions: false,
+        cpuMinVal: 1,
+        memMinVal: 1,
+        cpuMaxVal: 16,
+        memMaxVal: 32,
```

Comment:
@steve-gregory, curious about where this information should/might live:

- do we think that min & max values will vary on a provider to provider basis in a :cloud:? 
- will this vary on a "installation to installation" of Atmosphere basis? (diff between cyverse vs jetstream, etc)